spec/features/attachments/attachments_spec.rb

```diff
@@ -56,7 +56,7 @@
       expect(page).to have_text('Successful creation.')
       expect(page).to have_text(file_name)
 
-      click_link file_name
+      find('.work-package--attachments--files', text: file_name).click
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.